README.md

```diff
@@ -134,6 +134,14 @@ Get the request headers.
 
 Get the request body.
 
+#### .progress(loaded : number, total : number, lengthComputable : bool)
```

Comment:
thanks for the fix here. do you think this should have been implemented on the response interface?